cmd/agentless-scanner/awsutils/ebs.go

```diff
@@ -0,0 +1,464 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package awsutils
```

Comment:
Actually the `awsutils` name may be a bit misleading, since it only contains functions very specific to our use case, and not generic functions that are meant to be reused elsewhere.